packages/editor/src/components/post-visibility/index.js

```diff
@@ -133,6 +142,18 @@ export default function PostVisibility( { onClose } ) {
 	);
 }
 
+/**
+ * PostVisibilityChoice component.
+ *
+ * Renders a visibility choice option.
+ *
+ * @param {Object} props            - Component properties.
+ * @param {number} props.instanceId - Unique instance ID for the component.
+ * @param {string} props.value      - The value of the visibility option.
+ * @param {string} props.label      - The label for the visibility option.
+ * @param {string} props.info       - Additional information about the visibility option.
+ * @return {JSX.Element} The rendered component.
+ */
```

Comment:
This is not exported so we could leave it as is, though it doesn't hurt to have it 😄 .